omnigibson/prims/rigid_body_prim.py

```diff
@@ -0,0 +1,271 @@
+import math
+from typing import Literal
+
+import torch as th
+
+import omnigibson as og
+import omnigibson.lazy as lazy
+from omnigibson.utils.usd_utils import PoseAPI
+
+from .rigid_prim import RigidPrim
+
+
+class RigidBodyPrim(RigidPrim):
```

Comment:
nit: can we rename this to be RigidDynamicPrim? That way the distinction is clear between Kinematic <--> Dynamic, which also aligns with how omni docs describe the two